src/storage/src/client/upload_object/unbuffered.rs

```diff
@@ -221,13 +166,12 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn upload_object_unbuffered() -> Result {
+    async fn send_unbuffered_bytes() -> Result {
```

Comment:
nit: we never call `send()` so just `upload_object_bytes` ?

Reply:
Sure, done.